mocks/api/v2/barong/resource/users/activity/all/GET--limit=10&page=3.mock

```diff
@@ -3,6 +3,7 @@ access-control-allow-origin: *
 access-control-allow-methods: GET, POST, PUT, PATCH, DELETE
 access-control-allow-headers: Origin, X-Requested-With, Content-Type, Accept, Authorization
 Access-Control-Expose-Headers: total
+Access-Control-Allow-Headers: *
```

Comment:
POST, PUT, PATCH, DELETE and TRACE verbs, being state changing verbs, should have a CSRF token attached to the request
Please follow this instruction and correct for every request